integration-tests/minio/src/test/java/org/apache/camel/quarkus/component/minio/it/MinioTestResource.java

```diff
@@ -30,6 +30,7 @@ public class MinioTestResource implements QuarkusTestResourceLifecycleManager {
     public static final String CONTAINER_SECRET_KEY = "MINIO_SECRET_KEY";
     private final String CONTAINER_IMAGE = "minio/minio:RELEASE.2020-12-03T05-49-24Z";
```

Comment:
Might be good to update the container image. Maybe to this:

https://github.com/quarkiverse/quarkus-minio/blob/main/minio-client/deployment/src/main/java/io/quarkiverse/minio/client/deployment/devservices/MinioDevServicesBuildTimeConfig.java#L29

Reply:
done